### PR TITLE
Caching updates

### PR DESCRIPTION
**Pull request by loosebazooka**
merging of @thesayyn #1242 into main

### CODE REVIEW

**Comment by loosebazooka** [comment]
before merge: change cloud build to look for build file in .cloudbuild/cloudbuild.yaml

File: .cloudbuild/release.sh
Comment (thesayyn) [inline .cloudbuild/release.sh:15]: nit: turned out to be unnecessary. can be removed

**Review by thesayyn — APPROVED** [review]

File: .cloudbuild/release.sh
Comment (loosebazooka) [reply .cloudbuild/release.sh:15]: nice I can use that to retrigger the GHAs.

**Review by bobcallaway — APPROVED** [review]

**Comment by loosebazooka** [comment]
Once again. Do not merge till the cloud build trigger is updated

**Pull request merged**
